includes/Listeners/Commerce.php

```diff
@@ -25,6 +25,7 @@ public function register_hooks() {
 		// Paypal Connection
 		add_filter( 'pre_update_option_yith_ppwc_merchant_data_production', array( $this, 'paypal_connection' ), 10, 2 );
 		add_filter('update_option_ewc4wp_sso_account_status', array($this, 'ecomdash_connected'));
+		add_filter( 'woocommerce_update_product', array( $this, 'create_product' ), 100, 2 );
```

Comment:
We're using the `woocommerce_update_product` filter, but have a callback function called `create_product`.  We should probably be using the `woocommerce_new_product` filter here to avoid firing an event on all product changes.

Reply:
@ramyakrishnai I just saw the other notes from Evan. If we do want to track both created and updated products, then let's rename the `create_product` method to something like `product_created_or_updated` just to make things more clear.